client/my-sites/backup/backup-contents-page/file-browser/use-prepare-download.ts

```diff
@@ -36,11 +32,14 @@ export const usePrepareDownload = ( siteId: number ) => {
 		// Reset the status to not started so that the user can try again.
 		setStatus( PREPARE_DOWNLOAD_STATUS.NOT_STARTED );
 
-		// Dispatch an error notice
-		dispatch( onPreparingDownloadError() );
-	}, [ dispatch ] );
+		onError();
```

Comment:
It could come in handy at some point to pass on the `error` returned from `useQuery` but it's not a blocker at this point.